apps/system/test/marionette/audio_channel_belonged_to_system_test.js

```diff
@@ -0,0 +1,48 @@
+/* jshint node: true*/
+/* global marionette, setup, test*/
+'use strict';
+
+var System = require('./lib/system');
+var assert = require('assert');
+
+marionette('Audio channels belonged to system app', function() {
```

Comment:
nit: `Audio channels that belongs to system app` or `Audio channels owned by system app`